pages/calcentral/my_dashboard_my_classes_card.rb

```diff
@@ -59,7 +59,7 @@ def teaching_course_site_names
       # Returns a collection of course site descriptions displayed in the teaching section
       # @return [Array<String>]
       def teaching_course_site_descrips
-        teaching_course_site_desc_elements.map &:text
+        (teaching_course_site_desc_elements.map &:text).compact
```

Comment:
@johncrossman yes indeed. If you don't mind, I'll leave that as is since the entire My Dashboard UI will be dropped from this suite in a matter of weeks.